v2/pkg/subscraping/sources/github/github.go

```diff
@@ -38,18 +39,21 @@ type response struct {
 // Source is the passive scraping agent
 type Source struct{}
 
+var apiKeys []string
```

Comment:
this should be stored within source variable instead of as a global. and this is relevant for all sources

Reply:
done